custom_components/wellbeing/api.py

```diff
@@ -279,17 +281,17 @@ def get_appliance(self, pnc_id):
 
 class WellbeingApiClient:
 
-    def __init__(self, username: str, password: str, session: aiohttp.ClientSession) -> None:
+    def __init__(self, username: str, password: str, hass: HomeAssistant) -> None:
         """Sample API Client."""
         self._username = username
         self._password = password
-        self._session = session
         self._access_token = None
         self._token = None
+        self.hass = hass
```

Comment:
Is there a reason to not make `self.hass` private?

Reply:
No, that is a good idea. It should be private